src/client/components/Button/ButtonComponent.styles.css

```diff
@@ -0,0 +1,35 @@
+.button {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-weight: 700;
+  border: 0;
+  border-radius: 3em;
+  cursor: pointer;
+  display: inline-block;
+  line-height: 1;
+}
+
+.button--primary {
+  color: white;
+  background-color: #1ea7fd;
+}
+
+.button--secondary {
+  color: #333;
+  background-color: transparent;
+  box-shadow: rgba(0, 0, 0, 0.15) 0 0 0 1px inset;
+}
+
+.button--small {
```

Comment:
If you want others to specify if the button is small, medium or larger, these should be props to the button

Reply:
okay... i will remove these